app/commands/track/update_build_status.rb

```diff
@@ -213,11 +218,11 @@ def syllabus_health
   end
 
   def syllabus_volunteers
-    authors = User.with_attached_avatar.where(id: Concept::Authorship.where(concept: taught_concepts).select(:user_id)).
-      or(User.with_attached_avatar.where(id: Exercise::Authorship.where(exercise: active_concept_exercises).select(:user_id)))
+    authors = User.where(id: Concept::Authorship.where(concept: taught_concepts).select(:user_id)).
```

Comment:
I haven't yet split this up into two queries, as that has proven slightly tricky as the `serialize_volunteers` method is designed to work with two queries. I could duplicate some code, but it might be that with the change in the `CombineAuthorsAndContributors` class, the perf is better.

Reply:
Yeah probably. I'll have a think